wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/css/openverse.css

```
@@ -2,7 +2,8 @@
     display: block;
 
     width: 100%;
-    height: calc(100vh - var(--wp-global-header-height));
+    height: calc(100vh - var(--wp-global-header-height) - var(--wp-admin--admin-bar--height))
```

```suggestion
    height: calc(100vh - var(--wp-global-header-height) - var(--wp-admin--admin-bar--height, 0px))
```

Might be good to add a fallback there in case the admin bar gets disabled again.